### DISCUSSION TITLE
【求助】这是什么以骸？

### DISCUSSION THREAD

**Discussion by liu1145** [share121/inter-knot#1415] [崩坏：星穹铁道]
如题
在在̵̓͑͆͐͑̕空҉̓̔͊͊̽́̀̀̽͊͠洞̵͂̅̆̎͆̓͂̂͑̽̾͗͌̀̽̓͠乱̶͆̇͋̿͒̓̂͆́̋̈̀͒͛̿̚͠逛҈͐͑̂͌͛̀̓̉̌̄͐̔̑́͑̕的̷́̽̔́̇̌̾̑͆̋͝时҉̐̒͐͗̉͒̂̈͐͡候҈͛̽͛͗͗͆̿̆̔͆̈́͆̉̽͡，我发̸̈̈́́͋͒́͌̽̓̅͗̋͐̅͋͠现̶̃͗̓̇̾͐̈́̽͞了҉̉́͂̓̐̅̏͂͡一̴̔̽̂͐́̾̂́̋̾̿͋̌̓̚͝片̷̛̅͑̇̒̀͑̍͐̔̆͒͊́̚地҈̿̀̆͒̈́̽̒̄͒̾̇͞方̴̍͊̂̔̓̍͗̚͞冒̸̔͒̍̇͌̽̋̐̂̋͂́̋͝着̷̊̃̈̏͑̍̈̀͑̑̿́͆̽͡红̷̛̍̊̈́̔̈́̅̎̀́̅̒́͑͌͒光҉̓̋̀͒̐̋̅̂̿̀̾́͡，̸̌̅̀̎̈́͋̔̌̋̊̀̅̽̇͡，出҉̧͙͍̘̀̋͡于҈̡̱҇̈́好̶̡͙͎͈̿͠奇҉̧̦̗҇̇̊，我҉̪̈͛̆͜͝就҉̮̪̖͒̓̾͜͝去̷̨̳̬̲҇͐看҉̡̛̣̏了̵̡̟͍̉͠看҉̨̭̫̊̊̊̕，现̵在҉̷感҈觉̵̵精̸神҈̸非҈常҉好҈̴，̴̴，个҈̠͉̪̥̤̖͙̥͓͙̯̪̣̞̥̤͎͕̣̜̌̉̇̀̎̒̾̆̓͛̉̐̄͐͗̈͆́͛͗̀͒̚人҉̝͕̞͈̞̜͕̬̮͖̣̞̪͙͂͛̈́̈̾́͐̀̊̔͐̒̊̀́ͅ猜҉̙̗͎̘͓͖̝̬͕͈͓͈͔̋͗͂͆͆́͗͛̒̿̎͐̉́̈̀̈̈测̷̳͕͈̪͉͈̮͖͉͓͉͓̗̜̈͐̄́̎͛̍̾̿̑̊̏̓̓̒̌͌̑̈̏̚ͅ应̷̭̯͚̳̜̞̤̬̯̮͍͍̤̫͉̱̤͖̤̫̘́̾̅͑̄͛̍̈́̂̔̿́͌̓̐͛̅̒̀͒̅̾̆̚ͅͅ该҈͙̳̝̠̩͎̱͕̣̮̙̬̦͉͙͓͈̠̥̫͍͙̅͌̓̒͐̀̋͆͌̂̈́͑̑̿̓̈͛ͅͅ是̵̱͖͚͇͕̳͈̟͍͈̲͚̌̈́̐́̂̓͆̔̍͐̆͒̓̿̓͛́̃͋̈́̔͋有̴̝̪̝̳͕̗̯̫̖̜̤͇͖̗͓̩̖̒̾̓̓̅̌̊͑͛̉̓͋͒͆̓益҈̰̘͇̜̯̠̬̳̘̣̗̠̣̰̮̤̩̗̘̊̀̂̈̍̅͑͋͊̊͐̄͊́̐̄͑͐͛̔́͌̑的̶̙̬̭̞̯̮͖̥̫̳̱̬͊̀̔͆͆̐̽͐̋͌͋́̄̄ͅ，有̶̖̞̠̯͇͎̝̯͕̘͈͙̳̙͖͔͍͈̗̮͖̣͇͉͐͋͑̉͛̋̎͌̓̍̑̋̓̌͆͗̑͛͐̚̚人̵̘͓̩͕̪̮̮̜̟̟̦͉̰̦̝̤̘̰̝̲͎̱͇̊̈́͐͋͂̿̏͗͐̊͆͐̉̚能̷̠̳̪̙̮̖̖̦̦̘͙̗̳̘͈͚͎̙̱̱͋̔͑̽͂̍̿͋̆́̀̓̀̊̾̆͂́̉͛͐̓̌ͅ告҈̖̲̝̖͓͕̤͇̮̫̟̘͇̮̈̾́̅̊̎̓̾̅͒̌̂̓͑̚ͅ诉҈͉̤̤̦̗͍͙̠̙̤͙͇̟͇͊̒̉͌͗́̔̐̀̅̄̄̀͌̋̈̿͊̓͐̇̚我҈̟̯͖͇̜̞͍̳̘͎̲̗͉̘̯̘̗̗̜̎̂͛̀̉͊̋̒̎̀͛̽ͅͅ这҉͖̤͈͖̘̮͚̖͓̮̯̦͇̬̭͈͕͈̘̠̔̿̏̏̀̎̀͗́̔̊͌́̀̐̈́̅ͅ是҉̦̗͓͔̞̜̜̯͔͍̗̗͎̞̠͑́́́̓̊͒͊̏̽̽̏͒̀什̸̰̦̪̮̠͙͔̣͚̟̦̘̏͆̒͛̐̂̽̍̀͛͒̾͒ͅ么̸̗͓̯̩͚̣͈͍͚̖̜̫̝͋͆̉͌̑̄̅̀̀̋̒͒̃̍̂ͅ以҈̫̱͔̱͎̟̖̝̭̪͙͍̫̗̘̲̓̈́̈̓̇͆̔̂̊̈͆͐̎͌́̂͒̔̑̐̍ͅ骸̸̗͙̠̞͍̭̙̦͔̤̪͕͔͓̝̦͙̬̣͙͇̤͋͐͑͛̅̇̀̒̽̓͗̈́͆̃̃̔̔̈̓̚ͅ吗̶̝̳̗͓͚̙̫̬̣̪̝͕͍̙̈̄̍͊̓͛͒̓́̑̀͌̈̃͂̈̾̃̈́̌̄͑̔ͅ？҈̱̜̙͔͓̫̭̫͖̙̞̞͚͔͇̭̜͔̯͓̪̜͇͊̓̌͆͋͑̓̓̈̍̋̀̾̇͑͋̋̆̔̄͌̌̚ͅ，
![Uploading img_-1652221184_1712992766432.jpeg…]()
)

**Comment by liu1145** [comment]
如图
![img_-1652221184_1712992766432](https://github.com/user-attachments/assets/4057872d-5082-42b0-8277-d058ef9a20c5)

**Comment by WORLD-executeME** [comment]
lz走好

**Comment by obyz-wzr** [comment]
不出意外的话，哥们应该死了，致敬传奇绳匠

**Comment by AnimaVsk** [comment]
R.I.P~愿lz得以安息

**Comment by 1028-ou** [comment]
lz你还好吗
